app/models/departments.js

```diff
@@ -4,12 +4,119 @@
 const db      = require("../../app/db");
 const Promise = require("bluebird");
 
+const BaseModel = require("../lib/models").BaseModel;
+
 const DepartmentSupervisors = require("./departmentSupervisors");
 
+class Departments extends BaseModel {
+
+  constructor(data) {
+    super({
+      data: data,
+      columns: [
+        "department_id",
+        "organization",
+        "name",
+        "phone_number",
+        "created_by",
+        "active",
+        "created"
+      ]
+    })
+  }
+
+  static alterCase (departmentID, active) {
+    if (typeof active == "undefined") active = true;
+
+    return new Promise((fulfill, reject) => {
+      db("departments")
+        .where("department_id", departmentID)
+        .update({active: active})
+      .then(() => {
+        fulfill();
+      }).catch(reject);
+    });
+  }
 
+  static createOne (orgID, name, phoneNumber, userID) {
+    return new Promise((fulfill, reject) => {
+      db("departments")
+        .insert({
+          organization: orgID,
+          name: name,
+          phone_number: phoneNumber,
+          created_by: userID,
+          active: true
+        })
+      .then(() => {
+        fulfill()
+      }).catch(reject);
+    });
+  }
 
-// Class
-class Departments {
+  static editOne (departmentID, name, phoneNumber) {
+    return new Promise((fulfill, reject) => {
+      db("departments")
+        .where("department_id", departmentID)
+        .update({
+          name: name,
```

Comment:
Both are true. And yeah, I just made an update method in the voice branch for BaseModel, so might be worth it to compare notes and fix that later